l10n_th_withholding_tax_cert_multi/wizard/create_withholding_tax_cert.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2020 Ecosoft Co., Ltd (https://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+INCOME_TAX_FORM = [
```

Comment:
Please use this by import form here,
https://github.com/OCA/l10n-thailand/blob/13.0/l10n_th_withholding_tax_cert/models/withholding_tax_cert.py#L8

I.e.,
```
from odoo.addons.l10n_th_withholding_tax_cert.models.withholding_tax_cert import INCOME_TAX_FORM
```

Reply:
Edited.